RPIPs/RPIP-46.md

```diff
@@ -15,9 +15,9 @@ tags: tokenomics-2024, tokenomics-content
 ## Abstract
 Currently, the commission determines the payout of revenue split between rETH and each specific minipool. Other parties, such as RPL, gain value indirectly. This proposal allows for splitting revenue between four initial parties: rETH (the main product), node operators (the decentralized operators actually staking), voters (a subset of operators that have vote power), and a surplus revenue mechanism.
 
-This proposal also includes a small set of items for potential future use:
-- Seals that the security council can use to increase the NO share -- this is being used to find a reasonable setting based on the actual market.
-- An allowlist of controllers that may make changes to the settings, which allows for potential automation in the future
+This proposal also includes:
+1. A security council-controlled settings that can mildly increase the NO share -- this is being used to find a reasonable setting based on the actual market more quickly than the pDAO is able to act
```

Comment:
```suggestion
1. A setting controlled by the security council that can mildly increase the NO share -- this can be used to react to market conditions more quickly than the pDAO is able to act
```